pkg/k8s/apps/deployments.go

```diff
@@ -207,3 +207,13 @@ func (i *DeploymentApp) Destroy(ctx context.Context, c kubernetes.Interface) err
 func (i *DeploymentApp) PatchAnnotations(ctx context.Context, c kubernetes.Interface) error {
 	return deployments.PatchAnnotations(ctx, i.d, c)
 }
+
+// GetCloned Returns from Kubernetes the cloned deployment
+func (i *DeploymentApp) GetCloned(ctx context.Context, c kubernetes.Interface) (App, error) {
```

Comment:
Could we add a simple test for this function on deployment and statefulset?

Reply:
Yes, I can, but at the end is a simple wrapper of `statefulsets.Get`. Anyways, I'll add them